docs/notebooks/multiobjective_ehvi.pct.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+# # Multi-objective optimization with Expected HyperVolume Improvement Approach
+
+import gpflow
+import matplotlib.pyplot as plt
+import numpy as np
+import tensorflow as tf
+import math
+from util.plotting import plot_bo_points, plot_function_2d, plot_mo_history
+
+# +
+import trieste
+from trieste.acquisition.function import ExpectedHypervolumeImprovement
+from trieste.acquisition.rule import OBJECTIVE
+from trieste.models import create_model
+from trieste.models.model_interfaces import ModelStack
+from trieste.space import Box
+from trieste.data import Dataset
+from trieste.utils.mo_objectives import VLMOP2
+from trieste.utils.pareto import Pareto, get_reference_point
+
+np.random.seed(1793)
+tf.random.set_seed(1793)
+
+
+# -
+
+# ## Describe the problem
+#
+# In this tutorial, we provide multi-objective optimization example using the expected hypervolume improvement acquisition function.   
+# The synthetic function: VLMOP2 is a functions with 2 outcomes. We'll start by defining the problem parameters.
```

Comment:
We consider the VLMOP2 function ---  a synthetic benchmark problem with two objectives. We start by defining the problem parameters.